src/app/learn/get-started/advanced/include-static-assets/index.html

```diff
@@ -48,6 +45,28 @@
     </li>
   </ol>
 
+  <stache-page-anchor>
+    Include a static asset from a global style sheet
+  </stache-page-anchor>
+
+  <ol>
+    <li>
+      <p>
+        On the style sheet where you want to include the static asset, reference the static asset with a document-relative path <sky-code>assets/img/example.jpg</sky-code>.
```

Comment:
```suggestion
        On the style sheet where you want to include the static asset, reference the static asset with a document-relative path, such as <sky-code>assets/img/example.jpg</sky-code>.
```
Also, just checking: It's correct that this document-relative path does NOT have the `/` at the start like the root-relative path in the previous section did, right? That seems right, but just want to make sure.